pkg/branchcuts/bumper/replace_test.go

```diff
@@ -0,0 +1,70 @@
+package bumper_test
+
+import (
+	"testing"
+
+	"github.com/openshift/ci-tools/pkg/branchcuts/bumper"
+)
+
+func TestReplaceWithNextVersion(t *testing.T) {
+	tests := []struct {
+		name     string
+		line     string
+		major    int
+		expected string
+	}{
+		{
+			"Bump to the next version properly",
```

Comment:
Let's add the fields in all test cases to be more readable.

Reply:
Sure.